package.json

```diff
@@ -49,8 +49,8 @@
     "helmet": "7.0.0",
     "http-status-codes": "^1.4.0",
     "mixpanel": "^0.13.0",
-    "mongodb": "^3.5.9",
-    "mongoose": "^5.13.9",
+    "mongodb": "^5.9.1",
```

Comment:
Why we also have mongodb as dependency in this package 🤔 Isn't it a self dependency of mongoose ?